webpack.config.js

```diff
@@ -10,10 +10,23 @@ module.exports = {
           loader: 'babel-loader', // Allows Babel and Webpack to work together
           options: {
             presets: ['@babel/preset-react', '@babel/preset-env'] // Environment Presets... Not 100% what these do
+          },
+          {
+            test: [/\.css$/],
```

Comment:
not sure if this is what is causing you problems... but i have it written like this: `test: /\.css$/i`

Reply:
Good call, updated to be correct on my most recent PR.